component/integrations/node_exporter/node_exporter.go

```diff
@@ -0,0 +1,92 @@
+package node_exporter
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/go-kit/log"
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/metrics/scrape"
+	node_integration "github.com/grafana/agent/pkg/integrations/node_exporter"
+	"github.com/prometheus/common/model"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:    "integration.node_exporter",
+		Args:    node_integration.Config{},
+		Exports: Exports{},
+
+		Build: func(opts component.Options, args component.Arguments) (component.Component, error) {
+			return NewComponent(opts, args.(node_integration.Config))
+		},
+	})
+}
+
+// Exports are simply a list of targets for a scraper to consume
+type Exports struct {
+	Output []scrape.Target `river:"output,attr"`
```

Comment:
`output` feels a little weird here, even though it's consistent with targets.mutate. Should this be `targets` or `metrics`?